apps/ledger-live-desktop/src/renderer/components/WebPlatformPlayer/TopBar.jsx

```diff
@@ -128,7 +128,7 @@ const WebPlatformTopBar = ({
   config = {},
   webviewRef,
 }: Props) => {
-  const { name, icon } = manifest;
+  const { name, icon, displayBackToManifestApp } = manifest;
```

Comment:
I'm not a big fan of adding `displayBackToManifestApp` in the manifest and would actually prefer to have that behaviour by default if it makes sense